core/src/main/java/hudson/init/InitMilestone.java

```diff
@@ -82,7 +82,7 @@
      * By this milestone, all programmatically constructed extension point implementations
      * should be added.
      */
-    EXTENSIONS_AUGMENTED("Augmented all extensions"),
+    EXTENSIONS_AUGMENTED("Augmented all extensions"), // TODO nothing attains() this so when does it actually happen?
```

Comment:
Related only to the extent that while working on the initializer I was hoping to be able to use this milestone, but it appears bogus—while it will indeed happen between `PLUGINS_STARTED` and `JOB_LOADED`, it appears that nothing `attains` it specifically, so you cannot use this as a meaningful requirement for anything else.